Mathlib/Mathport/Syntax.lean

```diff
@@ -225,7 +225,6 @@ syntax generalizingClause := " generalizing" (ppSpace ident)+
 /- E -/ syntax (name := nthRwLHS) "nth_rw_lhs " num rwRuleSeq (location)? : tactic
 /- E -/ syntax (name := nthRwRHS) "nth_rw_rhs " num rwRuleSeq (location)? : tactic
 
-/- S -/ syntax (name := rwSearch) "rw_search" (config)? rwRuleSeq : tactic
 /- S -/ syntax (name := rwSearch?) "rw_search?" (config)? rwRuleSeq : tactic
```

Comment:
I think this one can also be dropped, since `rw?` has been implemented.

cc @semorrison

Reply:
Pushed a commit dropping this.